live-site/components/welcome-section/team-race/TeamRace.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { H3, P } from '../../../../shared-ui/style/typography';
+import { TeamProps, TeamRaceProps } from '../../../lib/types';
+import {
+  StyledRaceContainer,
+  StyledRaceContent
+} from '../WelcomeSection.styles';
+import Team from './Team';
+
+const TeamRace: React.FC<TeamRaceProps> = ({ teams }) => {
+  const orderTeams = (teams: TeamProps[]): TeamProps[] => {
+    return teams.sort((team1: TeamProps, team2: TeamProps) =>
+      team1.points > team2.points ? -1 : -1
+    );
+  };
+
+  return (
+    <>
+      <StyledRaceContainer>
+        <StyledRaceContent>
+          <H3>DesertRace</H3>
```

Comment:
This should be in white font
<img width="305" alt="Screenshot 2023-01-18 at 1 43 24 PM" src="https://user-images.githubusercontent.com/59738880/213267210-14832470-e447-4aa2-8eef-5ed9cbdc61d6.png">